lms/templates/edraak_university/university_id_success.html

```diff
@@ -0,0 +1,42 @@
+## mako
+
+<%inherit file="/main.html" />
+<%namespace name='static' file='/static_content.html'/>
+<%!
+    from django.utils.translation import ugettext as _
+    from django.core.urlresolvers import reverse
+    from django.conf import settings
+%>
+<%block name="bodyclass">view-in-course view-university-id-success</%block>
+
+<%block name="headextra">
+    <%static:css group='style-course-vendor'/>
+    <%static:css group='style-course'/>
+</%block>
+
+
+<%block name="pagetitle">${_("{course_number} University ID Successfully Saved").format(course_number=course.display_number_with_default) | h}</%block>
+<%block name="nav_skip">#course-university-id</%block>
+
+<%include file="/courseware/course_navigation.html" args="active_page='university_id'" />
+
+<section class="container activation">
+    <section class="message">
+        <h1 class="valid text-center">${_("Your University ID Has Been Saved")}</h1>
+
+        <hr class="horizontal-divider">
+            <p>
+                ${_('Your university ID have been successfully submitted and will be used for your grade records.')}
+            </p>
+
+            <p>
+                ${_('Should your university coordinator have any issues or amendments, he/she will contact you.')}
```

Comment:
Should your university coordinator have any comments on the information you entered, he/she will contact you.